src/Car.js

```diff
@@ -0,0 +1,60 @@
+import { Console, Random } from '@woowacourse/mission-utils';
+import { ERROR_MESSAGE } from './constant.js';
+import { errorString } from './util.js';
+
+class Car {
+  #name;
+  #position;
+
+  static MAX_CAR_NAME_LENGTH = 5;
+  static PRINT_POSITION_FORM = '-';
+
+  constructor(name) {
+    this.#validateName(name);
+
+    this.#name = name;
+    this.#position = 0;
+  }
+
+  #validateName(name) {
+    if (name.length === 0) {
+      throw new Error(errorString(ERROR_MESSAGE.MIN_CAR_NAME_LENGTH));
+    }
+
+    if (name.length > Car.MAX_CAR_NAME_LENGTH) {
+      throw new Error(errorString(ERROR_MESSAGE.MAX_CAR_NAME_LENGTH));
+    }
+  }
+
+  move() {
+    const canMove = this.#isMove();
+
+    if (canMove) {
+      this.#position += 1;
+    }
```

Comment:
이부분을 변수화하신 이유가 궁금합니다!
```suggestion
    if (this.#isMove()) {
      this.#position += 1;
    }
```
이렇게 축약할 수 있을 것 같아서요! 저는 개인적으로 이렇게만 해도 `isMove`인지 명확하게 알 수 있다고 생각합니당